database/src/migrations/20240202000001_observation_subcount_attribute_tables.ts

```diff
@@ -0,0 +1,193 @@
+import { Knex } from 'knex';
+
+/**
+ * Create new tables:
+ * - observation_subcount
+ * - subcount_critter
+ * - subcount_attribute
+ *
+ * Create new function/trigger:
+ * - tr_observation_subcount_count
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`--sql
+    -------------------------------------------------------------------------
+    -- Create tables
+    -------------------------------------------------------------------------
+    SET search_path = biohub;
+
+    CREATE TABLE observation_subcount(
+      observation_subcount_id    integer            GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+      survey_observation_id      integer            NOT NULL,
+      count                      integer            NOT NULL,
```

Comment:
Would you consider renaming this to `subcount`? It could make joins on `survey_observation` a bit easier to reason about.

Reply:
Yeah fair point. Updated.